src/Wallabag/CoreBundle/Resources/translations/messages.en.yml

```diff
@@ -144,6 +144,7 @@ entry:
         reading_time: 'estimated reading time'
         reading_time_minutes: 'estimated reading time: %readingTime% min'
         reading_time_less_one_minute: 'estimated reading time: <small class="inferieur">&lt;</small> 1 min'
+        number_of_tags: '{1}and one other tag|]1,Inf[and %count% other tags'
```

Comment:
Could you add that line into all others translated file but commented? It'll ease the translation.